foodnet/membership/tests.py

```diff
@@ -35,15 +47,15 @@ def setUp(self):
         email_address.save()
 
     def test_profile(self):
-        resp = self.client.get(reverse('profile'))
+        response = self.client.get(reverse('profile'))
```

Comment:
Very nice usage of mock factories but why do you always have to use your preference coding style? `resp` is as good as any other variable name.